recipes/ecell4_base/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "ecell4_base" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ecell/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 2e60e21e82c240739259c8a246c8889b9be86800420930eed39ba947d88ba7bc
+
+build:
+  number: 0
+  script_env:
+    - CC
+    - CXX
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python
+    - setuptools
+    - pybind11
+    - cmake
+    - hdf5
+    - gsl
+    - boost
+    - blas
```

Comment:
Please, just let ``- {{ compiler('cxx') }}`` and ``cmake`` to the build section.
The rest of the dependencies should be in the ``host`` section

Reply:
Thank you for the advice! I have just made a change.

Reply:
Thank you for the advice! I tried `libblas`. After some trials, I found that it still passes CI after removing `blas`. So I just removed it from here. Please check!